### PR TITLE
Transfers icon path to the emojis to define

### PR DESCRIPTION
**Pull request by NopemanMcHalt**
## About The Pull Request
This change shall allow to switch emoji set for the OOC, Dchat and Mime's PDA with only a define instead of changing 4 lines of code.

## Why It's Good For The Game
It's great for the modularization. Instead of writing `// BABABOOEY STATION CHANGE` for all 4 lines, it's now doable with only just one.

## Changelog
:cl: Anonymous
code: You can now change emoji set with just a define. See `__DEFINES/misc.dm`.
/:cl:

### CODE REVIEW

**Comment by Mothblocks** [comment]
why does this need to be a config and not just a `#define`?

**Comment by NopemanMcHalt** [comment]
> why does this need to be a config and not just a `#define`?

I didn't thought of that... Just wanted to make it modularizable for the downstreams to not touch original code at all.
If define will be better, I'll rewrite it.

File: code/__DEFINES/misc.dm [hunk 1]
```diff
@@ -551,3 +551,7 @@ GLOBAL_LIST_INIT(pda_styles, sortList(list(MONO, VT, ORBITRON, SHARE)))
 //while using the SKILLCHIP_RESTRICTED_CATEGORIES flag
 #define SKILLCHIP_CATEGORY_GENERAL "general"
 #define SKILLCHIP_CATEGORY_JOB "job"
+
+// Emojis icon set
+//Allows for easier swap
```
Comment (Mothblocks) [inline code/__DEFINES/misc.dm:556]:
```suggestion
/// Emojis icon set
```

Doc-comment, plus removes unnecessary comment.

**Review by Mothblocks — APPROVED** [review]

**Comment by NopemanMcHalt** [comment]
Might need label update.

**Comment by celotajstg** [comment]
Though it's a minor nitpick, I would recommend using emoji instead of emojis, as that's the correct plural.

**Review by Mothblocks — APPROVED** [review]

**Pull request merged**